### ISSUE TITLE
Custom basis and named algebras

### ISSUE THREAD

**Issue by tBuLi** [tBuLi/kingdon#16]
We would like to support custom bases, e.g. starting from `e0` for PGA's. 

Moreover, we would like the user to be able to provide custom basis, and to start an algebra simply from a name, e.g. "3DPGA".

Check with ganja.js and Clifford for their current settings.

**Comment by thejibz** [comment]
First of all, **THANK YOU** for your awesome work !

**TL;DR**: Allowing something like `locals().update(alg.cg_blades)` where `cg_blades` _just_ replace `e02` with `e20` in 2DPGA to fit for computer graphics (hence the "cg" prefix) where the "Y" axis starts from screen's top-left corner.

---

This is the convention in use at BiVector.net for example: https://bivector.net/tools.html?p=2&q=0&r=1
Or by Eric Lengyel overhere (though he starts index at 1, so e31 for e20): https://rigidgeometricalgebra.org/wiki/index.php?title=Rigid_Geometric_Algebra_for_2D_Space

Of course, this is not a top priority feature, more of an idea I got when using your lib to check the results of my own CG GA (toy)lib for Java.

Anyway, thank you again for sharing your work 👍

**Comment by tBuLi** [comment]
Fixed in kingdon v3.0.0 since today.
